test/controllers/AddAdditionalMetadataControllerSpec.scala

```diff
@@ -778,7 +778,7 @@ class AddAdditionalMetadataControllerSpec extends FrontEndTestHelper {
       actualFormField should equal(expectedField)
     }
 
-    "add the 'end_date' and 'lastModifiedDate' when present to the 'closureStartDate' field" in {
+    "add the 'end_date' only when it is available for the 'closureStartDate' field" in {
```

Comment:
Should there be a test to for showing the "last modified date" where there is no "end date"?